boardswarm-cli/src/main.rs

```diff
@@ -389,7 +389,7 @@ struct ConfigureArg {
 
 #[derive(Debug, Subcommand)]
 enum Command {
-    /// Configure boardswarm instances
+    /// Configure boardswarm instances access
```

Comment:
This change is handled in #23 so best to drop it from this PR to avoid conflicts later on